src/tests/architect_tests/test_state_table_generators.py

```diff
@@ -190,7 +190,7 @@ def test_sparse_states_from_query():
         engine = create_engine(postgresql.url())
         utils.create_binary_outcome_events(engine, 'events', input_data)
         table_generator = StateTableGeneratorFromQuery(
-            query='select entity_id from events where outcome_date < {as_of_date}',
+            query=f"select entity_id from events where outcome_date < '{as_of_date}'::date",
```

Comment:
You actually don't want the f-string here, I think otherwise this would work. As it is it complains about no `as_of_date` local var.

https://travis-ci.org/dssg/triage/builds/436312149

Reply:
By that I mean take out the f itself, otherwise this query string looks good.

Reply:
Oh, I see. You are right